util/big-decimal/src/bigdecimal.js

```diff
@@ -0,0 +1,9 @@
+import BigDecimal from 'big.js'
+
+BigDecimal.fromBigInt = (n, d) => {
```

Comment:
Not sure that it is good idea, because string or number may already be decimals. We can restrict this overload only for bigint type

```
BigDecimal(n: string | number | bigint)
BigDecimal(n: bigint, decimals: number | bigint)
BigDecimal(n: string | number | bigint, decimals?: number | bigint)
```